src/components/Modal.jsx

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import { motion, AnimatePresence } from "framer-motion";
+
+const Modal = ({
+  showModal,
+  toggleModal,
+  profileImages,
+  onHandleChange,
+  onAddProfileImage,
+}) => {
+  return (
+    <>
```

Comment:
React Fragment can be omitted, since `AnimatePresence` is the wrapping element already.